export/src/main/scala/uk/gov/nationalarchives/export/MetadataUtils.scala

```diff
@@ -74,15 +74,24 @@ class MetadataUtils(config: Config) {
            FROM "Consignment" c
            JOIN "Body" b ON b."BodyId" = c."BodyId"
            LEFT JOIN "Series" s ON c."SeriesId" = s."SeriesId"
-           WHERE  "ConsignmentId" = CAST(${consignmentId.toString} AS UUID) """
+           WHERE  "ConsignmentId" = CAST(${consignmentId.toString} AS UUID)
+           """
           .query[(String, String, String)]
           .unique
           .transact(transactor)
+      transferCompleteDate <-
+        sql""" SELECT TO_CHAR("TransferInitiatedDatetime",'YYYY-MM-DD HH:MI:SS') FROM "Consignment"
```

Comment:
Can you change it to 24-hour time `'YYYY-MM-DD HH24:MI:SS'` otherwise it will convert `2024-08-29 15:34:41` to `2024-08-29 03:34:41`?

Reply:
Good spot, thanks. I've changed it.